spec/acts_as_taggable_mongoid/taggable_spec.rb

```diff
@@ -219,20 +218,24 @@
     #   taggable.skill_list = "ruby"
     #   taggable.save
     #
-    #   expect(TaggableModel.tagged_with("ruby").group(:created_at).count.count).to eq(1)
+    #   expect(TaggableModel.tagged_with("ruby", on: :skills).group(:created_at).count.count).to eq(1)
     # end
 
-    # it "can be used as scope" do
-    #   taggable.skill_list = "ruby"
-    #   taggable.save
-    #
-    #   untaggable_model = taggable.untaggable_models.create!(name: "foobar")
-    #   scope_tag        = TaggableModel.tagged_with("ruby", any: "distinct", order: "taggable_models.name asc")
-    #   expect(UntaggableModel.joins(:taggable_model).merge(scope_tag).except(:select)).to eq([untaggable_model])
-    # end
+    it "can be used as scope" do
+      TaggableModel.create!(name: "Boby Jones", skill_list: "ruby")
+
+      taggable.skill_list = "ruby"
+      taggable.save!
+
+      expect(TaggableModel.tagged_with("ruby").count).to eq 2
+      expect(TaggableModel.tagged_with("ruby").where(name: "Bob Jones").count).to eq 1
+    end
+
+    it "should be able to find a tag using dates" do
+      Timecop.travel(2.days.ago) do
+        TaggableModel.create!(name: "Boby Jones", skill_list: "ruby")
+      end
```

Comment:
Add these back as best I could.

Added back a bunch of tests in this file from ActsAsTaggableOn.